src/PeoplePage/index.js

```diff
@@ -0,0 +1,69 @@
+import Icon from '../Icon';
+import LabelSmall from '../LabelSmall';
+import Avatar from '../Avatar';
+import React from 'react';
+import Typography from '../Typography';
+import classNames from 'classnames';
+import styles from './styles.css';
+import Wrapper from '../Wrapper';
+const PropTypes = React.PropTypes;
+
+function PeoplePage(props) {
```

Comment:
i would name the component to reference it exactly rather than its page, for example this one is Person item 1